### PR TITLE
Fix duplicate errors with fmt.Println(err) deletion

### LINKED ISSUE #8: Remove duplicate/ugly error messages
**Describe the bug**
Everytime an invalid input is given the error message shows up twice.  
Examples:
```
$ grofer invalid_command
Error: unknown command "invalid_command" for "grofer"
Run 'grofer --help' for usage.
unknown command "invalid_command" for "grofer"        # error message shows up again 


$  grofer proc -p 0

        pid no exist, done doggo a sad 
                \
                 \
                /^-----^\
                V  o o  V
                 |  Y  |
                  \ ⌓ /
                  / - \
                  |    \
                  |     \     )
                  || (___\====

Error: invalid pid
Usage:
  grofer proc [flags]

Aliases:
  proc, process, processess

Flags:
  -h, --help        help for proc
  -p, --pid int32   specify pid of process (default -1)

Global Flags:
      --config string   config file (default is $HOME/.grofer.yaml)

invalid pid                                   # error message shows up again
```

This seems to occur with every invalid input.

Moreover giving an argument of the wrong type will give an ugly error message, that is not meaningful to the user:
```
$ grofer proc -p notpid
Error: invalid argument "notpid" for "-p, --pid" flag: strconv.ParseInt: parsing "notpid": invalid syntax
Usage:
  grofer proc [flags]

Aliases:
  proc, process, processess

Flags:
  -h, --help        help for proc
  -p, --pid int32   specify pid of process (default -1)

Global Flags:
      --config string   config file (default is $HOME/.grofer.yaml)

invalid argument "notpid" for "-p, --pid" flag: strconv.ParseInt: parsing "notpid": invalid syntax       # error message shows up again and the message is ugly
```

**To Reproduce**
- Give any input that is invalid
- Give a string for the PID flag field

**Expected behavior**
Clean, unduplicated error messages

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Go Version: 1.14.6

### PR DESCRIPTION
**Pull request by tzzed**
# Description

Cobra already shows errors.
Duplicates errors are due to `fmt.Println(err)` in the `func Execute()` in the `root.go` file. 

Fixes #8 

Fixed by: `fmt.Println(err)` deletion.

### CODE REVIEW

**Review by Gituser143 — APPROVED** [review]
This seems to fix it, thanks for the PR! I'll wait for the other maintainers to approve before merging.

**Review by MadhavJivrajani — APPROVED** [review]
LGTM :smile:

**Review by metonymic-smokey — COMMENTED** [review]
I'll review this again once the Travis CI checks pass. The checks are not passing/failing right now so I'll wait till they pass.

**Comment by Gituser143** [comment]
@tzzed your PR was marked as abusive in our CI tests, could you please make a new PR from master to master? We'd appreciate it.

**Comment by MadhavJivrajani** [comment]
> @tzzed your PR was marked as abusive in our CI tests, could you please make a new PR to master? We'd appreciate it.

@tzzed will you be sending in your PR?

**Comment by tzzed** [comment]
I don't understand what you want sorry? What is the problem? Should I need redo a PR ?

**Comment by MadhavJivrajani** [comment]
For some unknown reason the PR was flagged as "abusive" by TravisCI, which is why it didn't pass the CI tests. 

If you could resend the same PR to the master branch once again from your existing branch on which you made changes, that'd be awesome!

Thanks so much 😄

**Pull request closed without merging**
